### ISSUE TITLE
MD006 only corrects first list item

### ISSUE THREAD

**Issue by nschonni** [igorshubovych/markdownlint-cli#70]
Given the following sample:

```md
### Playbooks

These playbooks are available to you:

  - **jenkins/host/create.yml**: Sets up jenkins ci hosts.

  - **jenkins/host/iptables.yml**: Update iptables rules so workers can connect.

  - **jenkins/worker/create.yml**: Sets up jenkins workers.

  - **jenkins/worker/upgrade-jar.yml**: Upgrades the worker jar file.

  - **jenkins/docker-host.yml**: Sets up a host to run Docker workers.
```

Only the first item is corrected with `--fix`, which actually changes the rendering of the list.

**Comment by DavidAnson** [comment]
This is because MD006 only ever reported the first violation. I think I looked briefly at this when adding fixInfo and it wasn’t trivial. But I didn’t realize a partial fix would change meaning. I’ll look into this for the next release.

**Comment by nschonni** [comment]
I think the ones that might cause problems if they aren't looked at as a whole list would be MD005, MD006, and MD007. MD027 might also have some list implications unless treated as one element.
I think stuff like remark-lint has to convert everything to an AST before it can re-write (but I don't really like the formatting it spits out)

**Comment by DavidAnson** [comment]
- MD006 behavior is updated to look at all items with https://github.com/DavidAnson/markdownlint/commit/3981bc7897dcaf05bd5c2fdadd08b4cdab04a459.
- MD007 is updated similarly with https://github.com/DavidAnson/markdownlint/commit/3e91da338cdbbddcc9b7172baa17c28188342dd8 and now includes auto-fix.
- MD005 already reported for all items.
- MD027 already handles all items.

The next release of `markdownlint` should resolve this issue once used `package.json` is updated.

**Comment by nschonni** [comment]
Awesome stuff @DavidAnson

**Comment by DavidAnson** [comment]
Fixed by bc10615c9acff25a2db45f2b82f26645cdea206f.
